portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,201 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/* Set up the content area grid */
+
+.content {
+  display: grid;
+  grid-template-rows: 55px 900px auto 270px;
+  grid-template-columns: auto;
+  grid-template-areas:
+    "header"
+    "intro"
+    "about"
+    "footer";
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  flex: 50%;
+}
+
+/* Reset default html margins and paddings */
+
+html, body { 
+  margin: 0; 
+  padding: 0;
+}
+
+html { 
+  scroll-behavior: smooth;
+}
+
+/* Style the text selection color */
+ 
+::selection {
```

Comment:
Apologies for the nit-pick (and for future reference we'll likely use 'nit' to signify nit-picks) but could you alphabetize these? Thanks!

Reply:
Will do! I tried my best to alphabetize but sometimes forgot